### DISCUSSION TITLE
[docs] broken link to "see also: request instances"

### DISCUSSION THREAD

**Discussion by jakkdl** [encode/httpx#3158] [General]
If you go to https://www.python-httpx.org/api/#client, scroll down to the documentation for `build_request` (being able to directly link the method would be nice...), and try to click on "See also: Request instances" you're hit with an error 404.

> build_request(self, method, url, *, content=None, data=None, files=None, json=None, params=None, headers=None, cookies=None, timeout=, extensions=None)
>
> Build and return a request instance.
>
>    The params, headers and cookies arguments are merged with any values set on the client.
>    The url argument is merged with any base_url set on the client.
>
> See also: [Request instances](https://www.python-httpx.org/advanced/#request-instances)

Skipping creating a discussion over this, as clearly you don't want broken links in your docs.

**Comment by alexdlukens** [comment]
Looks like the links were corrected in #3123 on Feb 29, but the docs have not been rebuilt since Feb 21 (#3095). A redeploy of the docs should fix this

**Reply by lovelydinosaur** [reply]
Well observed, thanks.
Our workflow is currently building and deploying the documentation on new releases.
